src/services/car/messageLead.ts

```diff
@@ -245,3 +246,75 @@ export const resendMessageLead = async ({
     result: {},
   }
 }
+
+export const fetchDealerWhatsappLeads = async ({
+  dealerId,
+  query,
+  options = {},
+}: {
+  dealerId: number
+  query: LeadQueryParams
+  options?: ApiCallOptions & { validateOnly?: boolean }
+}): Promise<Paginated<SearchWhatsappLead>> => {
+  const { validateOnly, ...otherOptions } = options
+  const { page, size, sort = {}, searchQuery } = query
+
+  const { sortOrder, sortType } = sort
+
+  const sortOrDefault = {
+    sortType: sortType || defaultLeadSort.sortType,
+    sortOrder: sortOrder || defaultLeadSort.sortOrder,
+  }
+
+  const queryParams = {
+    page: pageOrDefault(page, defaultDealerMessageLeadsPagination),
+    size: sizeOrDefault(size, defaultDealerMessageLeadsPagination),
+    sort: `${toCamelCase(sortOrDefault.sortType)},${toCamelCase(
+      sortOrDefault.sortOrder
+    )}`,
+    q:
+      searchQuery && searchQuery.length >= 3
+        ? encodeURIComponent(searchQuery)
+        : "",
+  }
+
+  const path = `dealers/${dealerId}/whats-app-tracking-entries?${toQueryString(
+    queryParams
+  )}`
+
+  return fetchPath({
+    path,
+    options: {
+      ...otherOptions,
+      isAuthorizedRequest: true,
+    },
+  })
+}
+
+export const hideWhatsappLead = async ({
+  dealerId,
+  whatsAppTrackingEntryId,
```

Comment:
I'd go with 
```suggestion
  id,
```
here. It's self explanatory from the context of the method name that it's the lead (or tracking entry) id